initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -313,6 +313,7 @@ configGenCmd() {
     # Creating logfiles for component
     logDir=$log/${compName}
 
+    export LSAN_OPTIONS=suppressions=${path}/etc/init.d/lsan-supp.txt
```

Comment:
This _could_ go ahead of other component uses, but there wasn't any other uses cases at the moment. We can move it (also add it) to other places if deemeed necessary later.

But this is specifically to address a leak in xmlStrndup in xalan as used by configgen, which I did not look like configgen's fault, but the 3rd party library - possibly would be fixed by using a more modern version